<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="1" ignored="0" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="69326" started-at="2021-05-06T21:39:16 IST" finished-at="2021-05-06T21:40:26 IST">
    <groups>
    </groups>
    <test name="Test" duration-ms="69326" started-at="2021-05-06T21:39:16 IST" finished-at="2021-05-06T21:40:26 IST">
      <class name="SeleniumVideos.SeleniumVideosPractice.AllureReportsPrac">
        <test-method status="PASS" signature="setup()[pri:0, instance:SeleniumVideos.SeleniumVideosPractice.AllureReportsPrac@654f0d9c]" name="setup" is-config="true" duration-ms="60408" started-at="2021-05-06T21:39:16 IST" finished-at="2021-05-06T21:40:17 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="loginTest()[pri:2, instance:SeleniumVideos.SeleniumVideosPractice.AllureReportsPrac@654f0d9c]" name="loginTest" duration-ms="5355" started-at="2021-05-06T21:40:17 IST" finished-at="2021-05-06T21:40:22 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [nopCommerce demo store123] but found [nopCommerce demo store. Login]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [nopCommerce demo store123] but found [nopCommerce demo store. Login]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at SeleniumVideos.SeleniumVideosPractice.AllureReportsPrac.loginTest(AllureReportsPrac.java:50)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="logoPresence()[pri:1, instance:SeleniumVideos.SeleniumVideosPractice.AllureReportsPrac@654f0d9c]" name="logoPresence" duration-ms="310" started-at="2021-05-06T21:40:24 IST" finished-at="2021-05-06T21:40:24 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoPresence -->
        <test-method status="SKIP" signature="regisTest()[pri:3, instance:SeleniumVideos.SeleniumVideosPractice.AllureReportsPrac@654f0d9c]" name="regisTest" duration-ms="3" started-at="2021-05-06T21:40:24 IST" finished-at="2021-05-06T21:40:24 IST">
          <exception class="org.testng.SkipException">
            <message>
              <![CDATA[Skipping the test]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.SkipException: Skipping the test
at SeleniumVideos.SeleniumVideosPractice.AllureReportsPrac.regisTest(AllureReportsPrac.java:58)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.SkipException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- regisTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:SeleniumVideos.SeleniumVideosPractice.AllureReportsPrac@654f0d9c]" name="tearDown" is-config="true" duration-ms="1665" started-at="2021-05-06T21:40:24 IST" finished-at="2021-05-06T21:40:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- SeleniumVideos.SeleniumVideosPractice.AllureReportsPrac -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
